contracts/WildcardSteward_v3.sol

```diff
@@ -576,24 +611,31 @@ contract WildcardSteward_v3 is Initializable {
         (transferSuccess, ) = recipient.call.gas(2300).value(_wei)("");
     }
 
+    // if credit balance exists,
+    // if amount owed > creidt
+    // credit zero add amount
+    // else reduce credit by certain amount.
+    // else if credit balance doesn't exist
+    // add amount to balance
+
     function _updateBenefactorBalance(address benefactor) public {
```

Comment:
This function now ensures that when updating benefactor balance, either debit or credit will always be zero.